src/Tizen.Multimedia.Vision/MediaVision/InferenceModelConfiguration.cs

```diff
@@ -246,12 +247,38 @@ public string CategoryFilePath
             }
         }
 
+        /// <summary>
+        /// Gets or sets the path of inference model's metadata file.
+        /// </summary>
+        /// <remarks>
+        /// This value should be set to use <see cref="ImageClassifier"/> or <see cref="ObjectDetector"/>.
+        /// </remarks>
+        /// <exception cref="ArgumentNullException">Metadata file path is null.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public string MetadataFilePath
+        {
+            get
+            {
+                return GetString(_keyMetadataFilePath);
+            }
+            set
+            {
+                if (value == null)
+                {
+                    throw new ArgumentNullException(nameof(value), "File path is null.");
+                }
+
+                Set(_keyMetadataFilePath, value);
+            }
+        }
+
         /// <summary>
         /// Gets or sets the inference model's mean value.
         /// </summary>
         /// <remarks>It should be greater than or equal to 0.</remarks>
         /// <exception cref="ArgumentOutOfRangeException">The value is invalid.</exception>
         /// <since_tizen> 6 </since_tizen>
+        [Obsolete("Deprecated since API9; Will be removed in API11. Please use MetadataFilePath instead.")]
```

Comment:
IMO a simple dot will look better in those messages than a semicolon.
```suggestion
        [Obsolete("Deprecated since API9. Will be removed in API11. Please use MetadataFilePath instead.")]
```

Reply:
Done